docs/admin/tutorial.md

```diff
@@ -1,53 +1,81 @@
 # Tutorial
 
```

Comment:
I have one. Maybe it would help to bold the 3 modes:
```suggestion
# Tutorial

This tutorial will show you how to use `bpfd`.
There are several ways to launch and interact with `bpfd` and `bpfctl`:

* **Basic Mode** - Run `bpfd` in foreground or background and straight from build directory.
  No `bpfd` or `bpfctl` users are created so always need `sudo` when executing `bptctl` commands.
  See [Basic Mode](#basic-mode).
* **`bpfctl` without `sudo`** - Run `bpfd` in foreground or background and straight from
  build directory, but also create the `bpfd` or `bpfctl` users and user groups.
  So if `bptctl` user group is added to a given user, `sudo` is no longer needed when executing `bptctl` commands.
  See [`bpfctl` without `sudo`](#bpfctl-without-sudo).
* **Systemd Service** - Run "bpfd" as a systemd service.
  See [Systemd Service](#systemd-service).

```

Reply:
I'll also rename:
"Basic Mode" as "Priviledged Mode"
"`bpfctl` without `sudo`" as "Unpriviledged Mode"